hipcheck/src/analysis/mod.rs

```diff
@@ -116,43 +104,6 @@ pub fn binary_analysis(db: &dyn AnalysisProvider) -> Result<QueryResult> {
 	})
 }
 
-pub fn churn_analysis(db: &dyn AnalysisProvider) -> Result<QueryResult> {
-	let results = db.churn_metric()?;
-	let value: Vec<F64> = results.commit_churn_freqs.iter().map(|o| o.churn).collect();
-	// @Todo - in RFD4 transition we lost the ability to flag commits, because
-	// the need to flag them as concerns is dependent on policy expr
-	Ok(QueryResult {
-		value: serde_json::to_value(value)?,
-		concerns: vec![],
-	})
-}
-
-pub fn entropy_analysis(db: &dyn AnalysisProvider) -> Result<QueryResult> {
-	let results = db.entropy_metric()?;
-	let value: Vec<F64> = results.commit_entropies.iter().map(|o| o.entropy).collect();
-	// @Todo - in RFD4 transition we lost the ability to flag commits, because
-	// the need to flag them as concerns is dependent on policy expr
-	Ok(QueryResult {
-		value: serde_json::to_value(value)?,
-		concerns: vec![],
-	})
-}
-
-pub fn identity_analysis(db: &dyn AnalysisProvider) -> Result<QueryResult> {
-	let results = db.identity_metric()?;
-	let num_flagged = results
-		.matches
-		.iter()
-		.filter(|m| m.identities_match)
-		.count() as u64;
-	let percent_flagged = num_flagged as f64 / results.matches.len() as f64;
-	let value = F64::new(percent_flagged).expect("Percent threshold should never be NaN");
-	Ok(QueryResult {
-		value: serde_json::to_value(value)?,
-		concerns: vec![],
-	})
-}
-
```

Comment:
You've removed identity analysis as a function, might as well remove the definition of `identity_metric()`